script/test/cmd/cmd_test.go

```
@@ -0,0 +1,354 @@
+	}
+	return path
+}
+
+// Kompose command
+func Kompose(command, provider, dockerComposeFile, result string) (cmd, output string) {
+	b, _ := ioutil.ReadFile(result)
+	file := strings.Split(dockerComposeFile, "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[1])
+		os.Exit(1)
+	}
+	cmd = string(out)
+	return cmd, output
+}
+
+// Kompose command for bundle
+func KomposeBundle(command, dockerComposeFile, result string) (cmd, output string) {
+	b, _ := ioutil.ReadFile(result)
+	file := strings.Split(dockerComposeFile, "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{"--bundle", dockerComposeFile, command, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[1])
+		os.Exit(1)
+	}
+	cmd = string(out)
+	return cmd, output
+}
+
+//GenerateArtifacts generates artifacts with -o command
+func GenerateArtifacts(command, provider, output_file, dockerComposeFile string) (outfile string) {
+	file := strings.Split(dockerComposeFile, "/")
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "-o", output_file, "-j"}
+	_, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[1])
+		os.Exit(1)
+	}
+	outfile = string(output_file)
+	return outfile
+}
+
+// CheckArtifacts checks the generated articfacts
+func CheckArtifacts(file string) error {
+	if _, err := os.Stat(file); err == nil {
+		fmt.Println("===> Starting Test ")
+		fmt.Println("Testing Generated Artifacts")
+		logrus.Info("\x1b[32;1mTest Pass\x1b[0m")
+	} else {
+		fmt.Println("\x1b[31;1m===> Test Fail <===\x1b[0m")
+		logrus.Error("\x1b[31;1mTest Fail\x1b[0m")
```

let's return the error here